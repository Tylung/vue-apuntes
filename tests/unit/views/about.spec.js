import { shallowMount } from  '@vue/test-utils'
import About from '@/views/AboutView'

describe( 'Pruebas con el About page', () => {
    test('Match con el snapshot', () => {
        const wrapper = shallowMount( About ) 
        expect( wrapper.html() ).toMatchSnapshot()
    })
})







