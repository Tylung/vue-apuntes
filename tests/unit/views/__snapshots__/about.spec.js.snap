// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con el About page Match con el snapshot 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
